pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -323,9 +331,19 @@ def on_train_epoch_start(self, trainer, pl_module):
         total_batches = total_train_batches + total_val_batches
 
         train_description = self._get_train_description(trainer.current_epoch)
+        if (
+            self.main_progress_bar_id is not None
+            and self._display_every_n_epochs
+            and (not trainer.current_epoch % self._display_every_n_epochs)
```

Comment:
so it's like if I set it to `2` it won't show at the 3rd, 5th, 7th epoch?

Reply:
It will show, but it won't start as a new bar